scrapy/http/headers.py

```diff
@@ -78,6 +79,12 @@ def values(self):
     def to_string(self):
         return headers_dict_to_raw(self)
 
+    def to_native_string_dict(self):
```

Comment:
Hm, method name is 'to_native_string_dict', but keys and values are converted to unicode - why is that?